docs/metrics.md

```diff
@@ -17,13 +17,25 @@ is a JSON-able dict.
 | Name | Type | Description |
 |:-------: |:-------: |:-------: |
 | categorical_accuracy | Task | Categorical Accuracy |
-| top_5_categorical_accuracy | Task | Top-5 Categorical Accuracy |
 | object_detection_AP_per_class | Task | Average Precision @ IOU=0.5 |
+| object_detection_class_precision | Task | Per-class precision @ IOU=0.5 |
+| object_detection_class_recall | Task | Per-class recall @ IOU=0.5 |
```

Comment:
I think we discussed omitting `object_detection_class_recall` and `object_detection_class_precision` since they were placeholders until mAP was implemented. If we are going to include them, we might want to indicate that they're computed at a score_threshold of 0.5

Reply:
Agreed - removing these from the documentation since AP is the metric we will typically care about.